recipe/meta.yaml

```diff
@@ -33,10 +33,10 @@ requirements:
     - typing_extensions
 
 test:
-  imports:
-    - datafusion
   commands:
     - pip check
+    - python -c 'from importlib.util import find_spec; assert find_spec("datafusion")'
+    - python -c 'import datafusion'                                                     # [build_platform != target_platform]
   requires:
     - pip
```

Comment:
This testing should work better when cross-compiling. Will allow us to reenable testing there